runtime/queries/rescript/textobjects.scm

```diff
@@ -0,0 +1,14 @@
+; Classes (modules)
+;------------------
+
+(module_declaration definition: ((_) @class.inner)) @class.outer
+
+; Functions
+;----------
+
+(function body: (_) @function.inner) @function.outer
+
+; Parameters
+;-----------
+
+(function parameter: (_) @parameter.inner @parameter.outer)
```

Comment:
You can remove parameter.outer/around here as it's not used and wild be differently queued if it were